third-party/realdds/include/realdds/topics/dds-topic-names.h

```diff
@@ -21,5 +24,179 @@ constexpr char const * NOTIFICATION_TOPIC_NAME = "/notification";
+        namespace key {
+            extern std::string const options;
+        }
+    }
+    namespace stream_header {
+        extern std::string const id;
+        namespace key {
+            extern std::string const type;
+            extern std::string const name;
+            extern std::string const sensor_name;
+            extern std::string const profiles;
+            extern std::string const default_profile_index;
+            extern std::string const metadata_enabled;
+        }
+    }
+    namespace stream_options {
+        extern std::string const id;
+        namespace key {
+            extern std::string const stream_name;
+            extern std::string const options;
+            extern std::string const intrinsics;
+            extern std::string const recommended_filters;
+        }
+        namespace intrinsics {
+            namespace key {
+                extern std::string const accel;
+                extern std::string const gyro;
+            }
+        }
+    }
+    namespace log {
+        extern std::string const id;
+        namespace key {
+            extern std::string const entries;
+        }
+    }
+    namespace query_options {
+        extern std::string const id;
+        namespace key {
+            extern std::string const option_values;
+        }
+    }
+}
+
+namespace control {
+    namespace key {
+        using notification::key::id;
+    }
+    namespace set_option {
+        extern std::string const id;
+        namespace key {
+            extern std::string const value;
+            extern std::string const option_name;
+            extern std::string const stream_name;
+        }
+    }
+    namespace query_option {
+        extern std::string const id;
+        namespace key {
+            using control::set_option::key::option_name;
+            using control::set_option::key::stream_name;
+        }
+    }
+    namespace query_options {
+        using notification::query_options::id;
+        namespace key {
+            using control::set_option::key::option_name;
+            using control::set_option::key::stream_name;
+            extern std::string const sensor_name;
+        }
+    }
+    namespace open_streams {
+        extern std::string const id;
+        namespace key {
+            extern std::string const stream_profiles;
+            extern std::string const reset;
+            extern std::string const commit;
+        }
+    }
+    namespace hwm {
+        extern std::string const id;
+        namespace key {
+            extern std::string const opcode;
+            extern std::string const param1;
+            extern std::string const param2;
+            extern std::string const param3;
+            extern std::string const param4;
+            extern std::string const data;
+            extern std::string const build_command;
+        }
+    }
+    namespace hw_reset {
+        extern std::string const id;
+    }
+}
+
+namespace reply {
```

Comment:
Previously we defined only `control` from client to server and `notification` from server to client. Now you separate `reply` for non-initalization notifications.
1. is it described properly in the documentation?
2. If the camera will propagate updated option values it will also be a reply, even if it isn't a reply to a query?

Reply:
Initialization is only through notifications.
Replies are notifications, too. But I wanted to separate the namespaces, so intellisense works better based on what you're doing.

The docs shouldn't describe this except by giving the key names. But the code should reflect the docs.

You'll notice `query-options` is both in `reply` and `notification`. That's because it can be either. When the camera periodically updates option values, it's a notification (no `control` or `sample`). In response to a `control`, it's a `reply`.